CHANGELOG.md

```diff
@@ -34,6 +34,7 @@ The first Node.js LTS release! See https://github.com/nodejs/LTS/ for details of
   - Added new `-c` (or `--check`) command line argument for checking script syntax without executing the code (Dave Eddy) [#2411](https://github.com/nodejs/node/pull/2411)
   - Added `process.versions.icu` to hold the current ICU library version (Evan Lucas) [#3102](https://github.com/nodejs/node/pull/3102)
   - Added `process.release.lts` to hold the current LTS codename when the binary is from an active LTS release line (Rod Vagg) [#3212](https://github.com/nodejs/node/pull/3212)
+  - Added new optional parameter `encoding` to `Buffer.indexOf`. Changed search algorithm from naive to naive + Boyer-Moore-Horspool. (Karl Skomski) [#2539](https://github.com/nodejs/node/pull/2539)
```

Comment:
No one's going to read that anyways, that's for sure. I'm feeling adding to the changelog after the release isn't quite right. Let's remove it.